packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -268,6 +269,10 @@ function useFocusContainment(scopeRef: RefObject<HTMLElement[]>, contain: boolea
       scope.forEach(element => element.removeEventListener('focusout', onBlur, false));
     };
   }, [scopeRef, contain]);
+
+  useEffect(() => () => {
```

Comment:
oh this is very hard to read. I thought it was happening on mount because I didn't see the double arrow function. Can we make it more clear?

```jsx
useEffect(() => {
  return () => {...};
}, []);

Reply:
haha, we can, but then eslint will yell ;)

Reply:
ugh, I dislike that rule